package.json

```diff
@@ -55,10 +57,11 @@
     "prettier": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier:fix": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\" --write",
     "start": "env-cmd --no-override -e dev,b2c,$SPA_ENV ng serve",
+    "start:local": "env-cmd --no-override -e local,b2c,$SPA_ENV ng serve --configuration development",
```

Comment:
Command `yarn start:local` is quite confusing, as the `yarn start` always starts the local Angular dev server. But you mean the local _backend_.

I'm not in favor of adding this script here. Instead, I advice to rely on passing a custom value of `$SPA_ENV` (extension point) in your development: 

e.g. `$ export SPA_ENV=local && yarn start`

Reply:
OK.